core/src/main/java/com/orientechnologies/orient/core/db/tool/ODatabaseExport.java

```diff
@@ -69,11 +73,22 @@ public ODatabaseExport(final ODatabaseDocumentInternal iDatabase, final String i
     }
     final File f = new File(fileName);
     if (f.getParentFile() != null)
+      //noinspection ResultOfMethodCallIgnored
       f.getParentFile().mkdirs();
     if (f.exists())
+      //noinspection ResultOfMethodCallIgnored
       f.delete();

```

Comment:
I suppose would be good to put warning here,  I do not think that tmp file may contain important information but would be better for user if he would be aware if something is removed.